frontend/apps/reader/modules/readerpaging.lua

```diff
@@ -908,8 +908,8 @@ function ReaderPaging:onGotoPageRel(diff)
     local x_pan_off, y_pan_off = 0, 0
     local right_to_left = self.ui.document.configurable.writing_direction and self.ui.document.configurable.writing_direction > 0
     local bottom_to_top = self.ui.zooming.zoom_bottom_to_top
-    local h_progress = 1 - self.ui.zooming.zoom_overlap_h / 100
-    local v_progress = 1 - self.ui.zooming.zoom_overlap_v / 100
+    local h_progress = 1 - self.ui.zooming.zoom_overlap_h * 0.01
+    local v_progress = 1 - self.ui.zooming.zoom_overlap_v * 0.01
```

Comment:
Why (here and elsewhere) use `* 0.01` instead of `* (1/100)` that you did for all others ?
It reads just a bit easier `* 1 per hundred`, `* 1 pour cent `, `* 1 per cent`, `*1 percent`.